scripts/single-node.sh

```diff
@@ -10,7 +10,7 @@ KEY_NAME="validator"
 KEYRING_BACKEND="test"
 COINS="1000000000000000utia"
 DELEGATION_AMOUNT="5000000000utia"
-CELESTIA_APP_HOME="${HOME}/.celestia-appd"
+CELESTIA_APP_HOME="${HOME}/.celestia-app"
```

Comment:
wow you're right `./celestia-app` looks correct b/c

Ref:
- https://github.com/celestiaorg/celestia-app/blob/159473dfbee3c2931cc34256b2471ff585cae5d7/app/app.go#L191
- https://github.com/celestiaorg/celestia-app/blob/159473dfbee3c2931cc34256b2471ff585cae5d7/app/app.go#L106

For some reason I thought `~./celestia-appd` was correct because locally I had a `~/.celestia-appd` directory. Since the rest of this script uses the environment variable `CELESTIA_APP_HOME`, it works before and after this change but I think we should merge it so that users who use this script don't get confused by two local directories.

Reply:
![django-unchained-drama](https://github.com/celestiaorg/celestia-app/assets/42654277/27a309cf-d210-4bf7-a4aa-2ba4a6a34a0d)

I've noticed this as well. I'm down to be consistent everywhere, however changing either would probably break a loooooooooot of stuff